### DISCUSSION TITLE
ESM configuration file format

### DISCUSSION THREAD

**Discussion by croraf** [eslint/eslint#14988] [General]
The ESLint documentation shows a list of **supported configuration file formats**. https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats

In the list the **JavaScript (ESM)** format is shown and it is explained how to use it. But in the last sentence it says 

> Note that ESLint does not support ESM configuration at this time.

I tried to set it up but I encounter some errors.

So my question is: Does ESLint support ESM configuration file format or not? (Please adjust this section of documentation to make it clear)

**Comment by aladdin-add** [comment]
it does not support.  `JavaScript (ESM)` most means: if you are working on an esm package, you will have to use the `.eslintrc.cjs` format.

oc, the docs can be improved. PRs are welcome! 😄
